app/controllers/volunteers_controller.rb

```diff
@@ -46,7 +46,7 @@ def create
 
   def edit
     authorize @volunteer
-    @supervisors = policy_scope current_organization.supervisors.active
+    @supervisors = policy_scope current_organization.supervisors.active unless @volunteer.has_supervisor?
```

Comment:
I don't think this functionality change belongs in a test-only change

Reply:
Not really a functionality change, the same condition added here is in the view as explained in the description... But removed.